src/language-js/parse/oxc.js

```diff
@@ -0,0 +1,84 @@
+import indexToPosition from "index-to-position";
+import { parseAsync as oxcParse, rawTransferSupported } from "oxc-parser";
```

Comment:
`oxc-parser` exports `parseSync()` as well.
You may not need to add `tryCombinationsAsync()` only for `oxc-parser`. 👀

Reply:
Thanks. But we prefer to use async functions when possible.